streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/ClientStateTest.java

```diff
@@ -37,126 +38,126 @@ public void shouldHaveNotReachedCapacityWhenAssignedTasksLessThanCapacity() thro
 
     @Test
     public void shouldHaveReachedCapacityWhenAssignedTasksGreaterThanOrEqualToCapacity() throws Exception {
-        client.assign(1, true);
+        client.assign(new TaskId(0, 1), true);
         assertTrue(client.reachedCapacity());
     }
 
 
     @Test
     public void shouldAddActiveTasksToBothAssignedAndActive() throws Exception {
-        client.assign(1, true);
-        assertThat(client.activeTasks(), equalTo(Collections.singleton(1)));
-        assertThat(client.assignedTasks(), equalTo(Collections.singleton(1)));
+        client.assign(new TaskId(0, 1), true);
```

Comment:
add a var `final TaskId tid = ...` (this should have been there for `int id = 1` already. Would improve the code base. Also for the other tests where it makes sense.